content/en/blog/_posts/2025-02-21-introducing-gateway-api-inference-extension/index.md

```diff
@@ -64,7 +65,7 @@ steps, e.g. extensions, in the middle. Here’s a high-level example of the requ
    and identifies the matching InferencePool backend.
 
 2. **Endpoint Selection**
-   Instead of simply forwarding to any pod, the Gateway consults an inference-specific routing extension. This
+   Instead of simply forwarding to any pod, the Gateway consults an inference-specific routing extension, e.g. ​endpoint selection extension. This
```

Comment:
maybe instead of 'e.g.'

> Instead of simply forwarding to any available pod, the Gateway consults an inference-specific routing extension - an ​endpoint selection extension - to pick the best of the available pods.

?

Reply:
Making the ^ change with one minor difference s/an ​endpoint selection extension/the Endpoint Selection Extension/